Makefile

```diff
@@ -48,21 +48,40 @@ NEWEST_GO_FILE = $(shell find $(SRC_DIRS) -name \*.go -exec $(STAT) {} \; \
 TYPES_FILES    = $(shell find pkg/apis -name types.go)
 GO_VERSION     = 1.8
 
+ALL_ARCH=amd64 arm arm64 ppc64le s390x
+
 PLATFORM?=linux
 ARCH?=amd64
 
+# TODO: Consider using busybox instead of debian
+ifeq ($(ARCH),amd64)
+	BASEIMAGE?=debian:jessie
+endif
+ifeq ($(ARCH),arm)
+	BASEIMAGE?=arm32v7/debian:jessie
+endif
+ifeq ($(ARCH),arm64)
+	BASEIMAGE?=arm64v8/debian:jessie
+endif
+ifeq ($(ARCH),ppc64le)
+	BASEIMAGE?=ppc64le/debian:jessie
+endif
+ifeq ($(ARCH),s390x)
+	BASEIMAGE?=s390x/debian:jessie
+endif
```

Comment:
should these be else-if's so that we can generate an error if they specify an ARCH we don't know about?